apps/web/app/(app)/onboarding/components/inapp/SurveyRole.tsx

```diff
@@ -0,0 +1,162 @@
+"use client";
+
+import { updateUserAction } from "@/app/(app)/onboarding/actions";
+import OnboardingTitle from "@/app/(app)/onboarding/components/OnboardingTitle";
```

Comment:
please use module exports instead of default exports to follow our code convention 😊:
```
import { OnboardingTitle } from "@/app/(app)/onboarding/components/OnboardingTitle";
```